mcs/class/Mono.Data.Sqlite/Test/SqliteTests.cs

```diff
@@ -38,17 +38,20 @@ namespace MonoTests.Mono.Data.Sqlite
 	[TestFixture]
 	public class SqliteTests
 	{
+		string _dataFolder;
 		string _databasePath;
 
 		[SetUp]
 		public void Setup ()
 		{
-			var dataFolder = Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.Personal), "SqlTest");
+			_dataFolder = Path.GetTempFileName ();
+			if (File.Exists (_dataFolder))
+				File.Delete (_dataFolder);
 
-			_databasePath = Path.Combine (dataFolder, "database.db");
+			_databasePath = Path.Combine (_dataFolder, "database.db");
 
-			if (!Directory.Exists (dataFolder)) {
-				Directory.CreateDirectory (dataFolder);
+			if (!Directory.Exists (_dataFolder)) {
+				Directory.CreateDirectory (_dataFolder);
```

Comment:
this can be simplified since _dataFolder is only used for constructing _databasePath, so we can just set _databasePath = Path.GetTempFileName () directly.